packages/@react-aria/actiongroup/src/useActionGroupItem.ts

```diff
@@ -29,7 +29,7 @@ const BUTTON_ROLES = {
   'multiple': 'checkbox'
 };
 
-export function useActionGroupItem<T>(props: ActionGroupItemProps, state: ListState<T>, ref: RefObject<HTMLElement>): ActionGroupItemAria {
+export function useActionGroupItem<T>(props: ActionGroupItemProps, state: ListState<T>): ActionGroupItemAria {
```

Comment:
breaking change?

Reply:
wasn't sure. I guess extra params are ignored. Not sure about TS. Otherwise we will have to suppress a lint warning about unused vars...

Reply:
Implementations using this will throw `Expected 2 arguments, but got 3.ts(2554)` 

This hook is not described in the ReactAria documentation, I guess it's not supposed to be used externally, so a breaking change will not hurt.